### PR TITLE
Mention that openelemety should be HA in production

### PR DESCRIPTION
**Pull request by viccuad**
## Test

To test this pull request, you can run the following commands:

### CODE REVIEW

File: versioned_docs/version-1.8/operator-manual/telemetry/opentelemetry/01-quickstart.md [hunk 1]
```diff
@@ -130,3 +130,9 @@ helm install --wait \
 
 We can now move to the next chapters to enable application metrics (via Prometheus
 integration) and application tracing (via Jaeger integration).
+
+:::note
+The OpenTelemetry operator must be ready, to create the needed sidecars
```
Comment (dgiebert) [inline versioned_docs/version-1.8/operator-manual/telemetry/opentelemetry/01-quickstart.md:135]: You should remove the comma after ready.

Comment (viccuad) [inline versioned_docs/version-1.8/operator-manual/telemetry/opentelemetry/01-quickstart.md:135]:
```suggestion
The OpenTelemetry operator must be ready to create the needed sidecars
```

Comment (viccuad) [reply versioned_docs/version-1.8/operator-manual/telemetry/opentelemetry/01-quickstart.md]: Thanks, done in https://github.com/kubewarden/docs/pull/288/commits/f46bac671fa043f8bb9aa06f41ff874a69a3c282/

**Comment by flavio** [comment]
thanks for the quick fix

**Pull request merged**